src/main/kotlin/lotto/model/game/Lotto.kt

```diff
@@ -1,35 +1,33 @@
-package lotto.model
+package lotto.model.game
 
-data class Lotto(private val lottoNumbers: Set<LottoNumber>) {
-    var lottoNums: Set<LottoNumber>
-        private set
+import lotto.model.result.Coincidence
+import lotto.model.input.Delimiter
 
+data class Lotto(val lottoNumbers: Set<LottoNumber>) {
     init {
         require(isUniqueSixNumbers(lottoNumbers)) { "로또 번호는 서로 다른 6개의 숫자여야만 합니다." }
-        lottoNums = lottoNumbers
     }
 
     constructor(numbers: Collection<Int>) : this(numbers.map(::LottoNumber).toSet())
 
-    constructor(stringNumbers: String) : this(Delimiter(stringNumbers).split().map { it.toInt() })
+    constructor(inputBeforeSplit: String) : this(Delimiter(inputBeforeSplit).split().map { it.toInt() })
 
-    fun getWinningCount(winningNumbers: Lotto): Int {
-        return lottoNums.count { winningNumbers.lottoNums.contains(it) }
+    fun getMatchCount(winningLotto: Lotto): Int {
+        return lottoNumbers.count { winningLotto.hasNumber(it) }
     }
 
-    fun hasBonusNumber(bonusNumber: LottoNumber): Boolean {
-        return lottoNums.contains(bonusNumber)
+    fun hasNumber(lottoNumber: LottoNumber): Boolean {
+        return lottoNumbers.contains(lottoNumber)
     }
 
     fun getResult(winningLotto: Lotto, bonusNumber: LottoNumber): Coincidence? {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    fun getResult(winningLotto: Lotto, bonusNumber: LottoNumber): Coincidence {
```